haystack/templates/pipeline.py

```diff
@@ -189,17 +125,6 @@ def from_predefined(cls, predefined_pipeline: PredefinedPipeline) -> "PipelineTe
         template_path = f"{TEMPLATE_HOME_DIR}/{predefined_pipeline.value}{TEMPLATE_FILE_EXTENSION}"
         return cls.from_file(template_path)
 
-    @classmethod
-    def from_url(cls, url: str) -> "PipelineTemplate":
```

Comment:
This is also something we don't want to teach to users, it can be problematic without an useful return